src/lib/network/wicked.rb

```diff
@@ -16,5 +16,27 @@ def reload_config(devs)
 
       SCR.Execute(BASH_PATH, "wicked ifreload #{devs.join(" ")}").zero?
     end
+
+    # Does an ifdown for each device named in devs
+    #
+    # @devs [Array] list of device names
```

Comment:
```rb
# @param devs [Array<String>] list of device names
```

I was pointing out that "@devs" is not a valid YARD tag, and this needs to be fixed 3 times in this file. The repetition is fine, no need for metamagic :)